README.md

```diff
@@ -1,6 +1,4 @@
-## Student projects at [European Spallation Source Data Management and Software Centre](https://europeanspallationsource.se/data-management-software-centre)
-* [Help us improve support for e.g. STEP or STL geometries](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD.pdf)
-* [Help us improve 3D visualisation in McStas](https://github.com/ess-dmsc-dram/student-projects/blob/main/project_CAD_mcdisplay.pdf)
+## Active student projects at [European Spallation Source Data Management and Software Centre](https://europeanspallationsource.se/data-management-software-centre)
```

Comment:
I am wondering if this means that these projects are already taken and being worked on, or does it mean they are available for someone to start?
If the latter, I would have left it as just "Student projects"

Reply:
These projects have already been worked on and a result (code) was achieved. So the thought was to leave them there "for reference", i.e more like  examples.

If a student wanted to work on a project in the same _theme_ I would probably find a solution, but as a project it would not become exactly what is in the PDF.

Reply:
Ok, so there are no new projects on the list that are available for the students to pick?

Reply:
Nope, not from my side at least. At home in bed and completely drained batteries atm.